packages/backend/src/api/resources/SamlConnection.ts

```diff
@@ -0,0 +1,63 @@
+import type { AttributeMappingJSON, SamlConnectionJSON } from './JSON';
+
+export class SamlConnection {
+  constructor(
+    readonly id: string,
+    readonly name: string,
+    readonly domain: string,
+    readonly idpEntityId: string | null,
+    readonly idpSsoUrl: string | null,
+    readonly idpCertificate: string | null,
+    readonly idpMetadataUrl: string | null,
+    readonly idpMetadata: string | null,
+    readonly acsUrl: string,
+    readonly spEntityId: string,
+    readonly spMetadataUrl: string,
+    readonly active: boolean,
+    readonly provider: string,
+    readonly userCount: number,
+    readonly syncUserAttributes: boolean,
+    readonly allowSubdomains: boolean,
+    readonly allowIdpInitiated: boolean,
+    readonly createdAt: number,
+    readonly updatedAt: number,
+    readonly attributeMapping: AttributeMapping | null,
```

Comment:
This property will always be present
```suggestion
    readonly attributeMapping: AttributeMapping,
```